platform/machine/qemu/disk.go

```diff
@@ -195,3 +193,12 @@ func setupDisk(additionalOptions ...string) (*os.File, error) {
 
 	return os.OpenFile(dstFileName, os.O_RDWR, 0)
 }
+
+func mkpath(basedir string) (string, error) {
```

Comment:
Given this function isn't exported or used anywhere else should we just drop the `basedir` parameter and revisit it if we decide to allow the setting of `TempDir` locations later?

Reply:
The next commit uses the argument.

Reply:
agh, grepping around fails me again :(